### PR TITLE
Fix skip actions

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: lib/gooddata/lcm/lcm2.rb [hunk 1]
```diff
@@ -299,9 +294,12 @@ def perform(mode, params = {})
 
         skip_actions = (params.skip_actions || [])
         actions = actions.reject do |action|
-          skip_actions.include?(action.to_s)
+          skip_actions.include?(action.name.split('::').last)
```
Comment (yenkins) [inline lib/gooddata/lcm/lcm2.rb]: Calls 'action.name' 2 times (DuplicateMethodCall)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/648/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/436/ : SUCCESS in 34s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/744/ : SUCCESS in 39s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/242/ : SUCCESS in 18s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/437/ : SUCCESS in 31s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/745/ : SUCCESS in 1m 49s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1158/ : FAILURE in 49m 29s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/916/ : SUCCESS in 32m 15s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/243/ : FAILURE in 13s
- gooddata-ruby-pronto-code-review-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED
- gooddata-ruby-integration-tests-zuul : SKIPPED
- gooddata-ruby-lcm-integration-tests-zuul : SKIPPED

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/246/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/440/ : SUCCESS in 1m 56s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/748/ : SUCCESS in 2m 05s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1161/ : FAILURE in 3m 12s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/919/ : UNSTABLE in 1m 21s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/247/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/441/ : SUCCESS in 1m 53s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/749/ : SUCCESS in 1m 49s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1162/ : FAILURE in 48m 42s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/920/ : SUCCESS in 31m 20s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/248/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/442/ : SUCCESS in 1m 50s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/750/ : SUCCESS in 39s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1163/ : FAILURE in 49m 15s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/921/ : SUCCESS in 36m 49s

**Comment by panjan** [comment]
extended test - staging

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/651/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1164/ : FAILURE in 45m 31s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/922/ : SUCCESS in 34m 00s

**Pull request merged**